### DISCUSSION TITLE
How I can create dropdown category in Discussions?

### DISCUSSION THREAD

**Discussion by rodukov** [community/community#33986] [Discussions]
I want to create category dropdown like this?
![image](https://user-images.githubusercontent.com/101352977/192050330-5d9ec94a-751a-4fca-af53-5427d0b75808.png)

**Thanks for the help! ❤️**

**Comment by agnelnieves** [comment]
Got the same question, was looking at the docs for like an hour trying to find this. But there's no mention for sub-categories or category sections, or drop-downs for that matter.

**Comment by LindseyB** [comment]
This isn't available outside of github.com/community presently.
